etc/config/compiler-explorer.amazon.properties

```diff
@@ -8,6 +8,7 @@ compiler-wrapper=./c-preload/compiler-wrapper
 max-asm-size=67108864
 staticMaxAgeSecs=30
 cacheConfig=InMemory(25);S3(storage.godbolt.org,cache,us-east-1)
+executableCacheConfig=S3(storage.godbolt.org,executableCache,us-east-1)
```

Comment:
I think we can use the same location as the regular cache: the setup means that things get autodeleted after 24 hours. If we feel there's a need to keep them separate, we need to make a change to the terraform config to set a TTL of 24 hours for `executableCache` locations.

Reply:
I'm certain: the caches are cryptographically secure. If we did find a collision we'd be eligible for some kind of prize in the crypto community :)

Reply:
Haha, no I meant the name of the object having a collision since they are hashed from the same source

Reply:
Just confirmed, they are indeed the same, so we can't store them in the same bucket unless we change something about the key or hash before storing. I'll add a _exec suffix.

Reply:
Oh! I see - I didn't notice the hash key could be the same! The suffix is great (as is changing the hash by adding `executable: true` or something :) Please feel free to merge :)